### DISCUSSION TITLE
Switch between Notes

### DISCUSSION THREAD

**Discussion by GSTbyMM** [saber-notes/saber#1210] [Ideas]
FOA, thank you so much for developing such an awesome note-taking app.

The only feature missing is the functionality to switch between different open notes. Suppose, I am teaching from a Book PDF and at the same time want to illustrate but then I have to exit from the full screen, go back, select the note/pages (on which I will explain and illustrate) and then full screen again. Have to click so many times.

Please provide a feature to set a "Shortcut Key" to press to switch between notes being in full-screen mode.

It will save a lot of time and look professional while teaching.

Thanks in Advance.

**Comment by akimon658** [comment]
As a tablet user, I think it would be nice to have not only a shortcut key but a tabbed UI like Goodnotes.

**Comment by ceskyDJ** [comment]
I think a little better place where to propose new features is Issues section. Others can vote for your proposal by their ”thumbs up“ reactions and the feature request will get higher priority in published list of top issues.
